client/app/telecom/telephony/line/management/mgcpIpRestriction/translations/Messages_fr_FR.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translations>
+   <translation id="telephony_line_mgcp_ip_restriction_title" qtlid="336831">Restriction MGCP par IP</translation>
+   <translation id="telephony_line_mgcp_ip_restriction_line_title" qtlid="390669">Restriction MGCP de la ligne</translation>
+   <translation id="telephony_line_mgcp_ip_restriction_line_info" qtlid="390682">Cette restriction est utilisée sur la ligne actuellement sélectionnée.</translation>
```

Comment:
It's better to say "La restriction TEST est utilisée sur la ligne actuellement ..."

More context, fewer questions :) #a11y